BEIMA.Backend.Test/StorageService/AzureStorageProviderTest.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using BEIMA.Backend.StorageService;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace BEIMA.Backend.Test.StorageService
+{
+    [TestFixture]
+    public class AzureStorageTest : UnitTestBase
+    {
+        private readonly IStorageProvider _storage;
+
+        public AzureStorageTest()
+        { 
+            var services = new ServiceCollection();
+            services.AddSingleton<IStorageProvider, AzureStorageProvider>();
+            var serviceProivder = services.BuildServiceProvider();
+            _storage = serviceProivder.GetRequiredService<IStorageProvider>();
+        }
+
+        [Test]
+        public void SmokeTest()
+        {
+            Assert.IsNotNull(_storage);
+            Assert.IsInstanceOf(typeof(AzureStorageProvider), _storage);
+        }
+
+        [Test]
+        public async Task GetFileStream_FileExists()
```

Comment:
We have been using the pattern of InitialState_Action_ResultingState() for test naming. For example, on this I would suggest using something like FileInStorage_GetFileStream_FileExists() or whatever makes more sense.